Description: Patch Cargo.toml to ensure dependencies are built locally
 This patch excludes the vendor/ directory from the cargo workspace
 to prevent various build issues and patches slint and
 supergfxctl to ensure they do not pull from remote repos
 .
 asusctl (6.0.12) oracular; urgency=medium
 .
   * New upstream release.
Author: Mitchell Augustin <mitchell@mitchellaugustin.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2024-12-13

--- asusctl.orig/Cargo.toml
+++ asusctl/Cargo.toml
@@ -34,6 +34,9 @@ default-members = [
     "cpuctl",
     "rog-control-center",
 ]
+exclude = [
+    "vendor/",
+]

 [workspace.dependencies]
 tokio = { version = "^1.39.0", default-features = false, features = [
@@ -97,3 +100,10 @@ opt-level = 3
 version = "1"
 default-features = false
 features = ["user-hooks"]
+
+[patch.'https://github.com/slint-ui/slint']
+slint = { version = "1.8" }
+slint-build = { version = "1.8" }
+
+[patch.'https://gitlab.com/asus-linux/supergfxctl']
+supergfxctl = { version = "5.2.4" }
